services/quota_manager/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Dockerfile for VPC Manager
+
+FROM jboss/base-jdk:11
+
+MAINTAINER Zhonghao Lyu <zlyu@futurewei.com>
```

Comment:
@Gzure Please update yourself as the maintainer of this file :-)

Also line 1, it is
"#Dockerfile for Quota Manager"